### PR TITLE
Dash/component spec

### PR DESCRIPTION
**Pull request by fanuntez**
Created basic demos for HTMLComponent.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

File: samples/dashboards/html-component/nested-elements/demo.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<script src="https://code.highcharts.com/dashboards.js"></script>
```
Comment (karolkolodziej) [inline samples/dashboards/html-component/nested-elements/demo.html:1]: Please pull the latest changes from the `master` and update that path to:
 `<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>`

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by pawellysy — APPROVED** [review]
LGTM!

File: samples/dashboards/html-component/single-element/demo.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<script src="https://code.highcharts.com/dashboards.js"></script>
```
Comment (karolkolodziej) [inline samples/dashboards/html-component/single-element/demo.html:1]: Here you use the same script ;)

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Also what should be the outcome of this demo?
I get: 
image

File: samples/dashboards/html-component/single-element/demo.html (hunk 2 above)
Comment (fanuntez) [reply samples/dashboards/html-component/single-element/demo.html]: Whoops, forgot that one.

**Comment by fanuntez** [comment]
> Also what should be the outcome of this demo? I get: image

It's supposed to be two images inside a div. [(Asana task)](https://app.asana.com/0/1200485006652765/1204494005141247)

https://github.com/highcharts/highcharts/assets/107555454/f30e5f46-7371-4a34-8330-8bd3155fbadf

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**